kalite/coachreports/features/environment.py

```diff
@@ -1,4 +1,4 @@
 """
 environment.py specific to the this app
 """
-from kalite.testing.base_environment import before_all, after_all, before_feature, after_feature
+from kalite.testing.base_environment import *
```

Comment:
In that case, I would say the problem here is that the module being imported from is missing an ```__all__```.